### PR TITLE
Drop unnecessary path prefix and export-ignore completion

### PR DESCRIPTION
**Pull request by raphaelstolt**
Completed the list of files to exclude from releases/distributions. The `LICENSE` file might be excluded from that list of files, but that's up to your liking. 

I'm also not quite sure if the `Makefile` is also required for the package itself in which case it should be excluded from that list.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,7 +1,12 @@
-/test               export-ignore
-/.coveralls.yml     export-ignore
-/.gitattributes     export-ignore
-/.gitignore         export-ignore
-/.php_cs            export-ignore
-/.travis.yml        export-ignore
-/phpunit.xml.dist   export-ignore
+.coveralls.yml     export-ignore
```
Comment (kelunik) [inline .gitattributes]: `.coverally.yml` should just be deleted.

File: .gitattributes [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+.editorconfig      export-ignore
+.gitattributes     export-ignore
+.gitignore         export-ignore
+.php_cs.dist       export-ignore
+.travis.yml        export-ignore
+CONTRIBUTING.md    export-ignore
+LICENSE            export-ignore
```
Comment (kelunik) [inline .gitattributes]: Please unignore `LICENSE`.

**Comment by kelunik** [comment]
@raphaelstolt Ping, could you update the PR?

**Comment by raphaelstolt** [comment]
@kelunik Done, the build is [currently](https://travis-ci.org/amphp/cache/jobs/360664165) red due to a failing `php-cs-fixer` lint in the vendor dir (???). I haven't looked any further into it than the last commit. Feel free to commit a fix into this pull request.

**Comment by kelunik** [comment]
Thanks, manually merged your first commit!

**Pull request closed without merging**